boltons/mathutils.py

```diff
@@ -0,0 +1,44 @@
+"""This module provides useful math functions on top of Python's built-in :mod:`math` module.
+"""
+
+
+def ceil_from_list(lst, x, allow_equal=True):
```

Comment:
why `_from_list` ? it actually allows any iterable   .  And  that probably means it would better fit in the `itertools` module.